src/js/components/TextInput/index.d.ts

```diff
@@ -23,6 +23,7 @@ export interface TextInputProps {
     suggestionIsOpen?: string;
   };
   name?: string;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
hi buddy, isn't it the default type of `onChange` by React? we try not to override the react types when unnecessary.

Reply:
So, It's exist in React for sure. However look here:
https://codesandbox.io/s/dreamy-field-0nhrg?file=/src/App.tsx

If I try to use the `TextInputProps` as a type for one object Key, this doesn't work.

Reply:
I figured-out that might be a issue where the `Omit` with the HTML Event stuff is being declared in the `declare const TextInput`

So when you import the `TextInputProps` interface that is being declared above. This doesn't have this `Omit....`